src/ios/Common/OSFANLConsentHelper.swift

```diff
@@ -0,0 +1,66 @@
+import FirebaseAnalytics
+import FirebaseCore
+
+@objc enum ConsentTypeRawValue: Int {
+    case adPersonalization = 1
+    case adStorage = 2
+    case adUserData = 3
+    case analyticsStorage = 4
+    
+    var stringValue: String {
+        switch self {
+        case .adPersonalization: return "ad_personalization"
+        case .adStorage: return "ad_storage"
+        case .adUserData: return "ad_user_data"
+        case .analyticsStorage: return "analytics_storage"
+        }
+    }
+}
+
+@objc enum ConsentStatusRawValue: Int {
```

Comment:
Instead of adding a new `stringValue` variable, we should conform the enum to `CustomStringConvertible` and implement the `description` property.